processor/deltatocumulativeprocessor/README.md

```diff
@@ -28,3 +28,15 @@ processors:
 ```
 
 There is no further configuration required. All delta samples are converted to cumulative.
+
+## Troubleshooting
+
+The following metrics are recorded when [telemetry is
+enabled](https://opentelemetry.io/docs/collector/configuration/#telemetry):
+
+| Name                                     | Description                                                                           | Unit    |
+|------------------------------------------|---------------------------------------------------------------------------------------|---------|
+| `deltatocumulative_streams_count`        | Number of streams currently tracked by the aggregation state                          |         |
+| `deltatocumulative_datapoints_processed` | Total number of datapoints processed, whether successful or not                       |         |
+| `deltatocumulative_datapoints_dropped`   | Faulty datapoints that were dropped due to the reason given in the `reason` attribute |         |
+| `deltatocumulative_seconds_lost`         | Total length of all gaps in the streams, which occur e.g. due to lost in transit      | seconds |
```

Comment:
Question: What is our intended use-case for this metric? IE, why does a user care if there are gaps or not? They should be able to directly see the gaps in the metrics themselves

Reply:
And also, why are these gaps considered "lost seconds"? A "normal" gap would arise from a process restart. Which isn't "lost" data

Reply:
As a collector maintainer I would rather have those metrics to know that this processor works as intended instead of waiting that some dashboard user would notice a gap, decides to care enough to approach a collector maintainer, and then figuring out that the issue is in deltatocumulative processors.

Reply:
```
d0: start: t0, time: t1
d1: start: t1, time: t2
d2: start: t2, time: t3
```

If those are all part of the same data stream, wouldn't they all have `start: t0`? IE, the time when the datastream was "created"? Perhaps I'm misunderstanding the spec?

Reply:
Ahhhh. Good find. Thanks for clearing that up :)